### PR TITLE
Docsup 9474: document table comments

### PR DESCRIPTION
**Pull request by kirillikoff**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #23548

### CODE REVIEW

File: docs/en/sql-reference/statements/create/table.md [hunk 1]
```diff
@@ -354,3 +354,70 @@ SELECT * FROM base.t1;
 │ 3 │
 └───┘
 ```
+
+## COMMENT TABLE {#comment-table}
+
+You can add a comment to the table when you creating it.
+
+!!!note "Note"
+    The comment is supported for all table engines besides [Kafka](../../../engines/table-engines/integrations/kafka.md) and [RabbitMQ](../../../engines/table-engines/integrations/rabbitmq.md).
```
Comment (olgarev) [inline docs/en/sql-reference/statements/create/table.md:363]:
```suggestion
!!!note "Note"
    The comment is supported for all table engines except [Kafka](../../../engines/table-engines/integrations/kafka.md) and [RabbitMQ](../../../engines/table-engines/integrations/rabbitmq.md).
```

File: docs/en/sql-reference/statements/create/table.md [hunk 2]
```diff
@@ -354,3 +354,70 @@ SELECT * FROM base.t1;
...
+COMMENT 'Comment';
+```
+	
+**Example**
+
+Query:
+
+``` sql
+CREATE TABLE t1 (x String) ENGINE = Memory COMMENT 'The temporary table';
+SELECT name,comment FROM system.tables WHERE name = 't1';
```
Comment (olgarev) [inline docs/en/sql-reference/statements/create/table.md:383]:
```suggestion
SELECT name, comment FROM system.tables WHERE name = 't1';
```

File: docs/en/sql-reference/statements/create/table.md [hunk 3]
```diff
@@ -354,3 +354,39 @@ SELECT * FROM base.t1;
 │ 3 │
 └───┘
 ```
+
+## COMMENT TABLE {#comment-table}
```
Comment (olgarev) [inline docs/en/sql-reference/statements/create/table.md:358]:
```suggestion
## COMMENT Clause {#comment-table}
```

File: docs/en/sql-reference/statements/create/table.md [hunk 4]
```diff
@@ -354,3 +354,39 @@ SELECT * FROM base.t1;
...
+
+**Syntax**
+
+``` sql
+CREATE TABLE db.table_name
+(
+    name1 type1, name2 type2, ...
+)
+ENGINE = engine
+COMMENT 'Comment';
```
Comment (olgarev) [inline docs/en/sql-reference/statements/create/table.md:374]:
```suggestion
COMMENT 'Comment'
```

File: docs/ru/sql-reference/statements/create/table.md [hunk 5]
```diff
@@ -346,4 +346,39 @@ SELECT * FROM base.t1;
...
+	
+**Синтаксис**
+
+``` sql
+CREATE TABLE db.table_name
+(
+    name1 type1, name2 type2, ...
+)
+ENGINE = engine
+COMMENT 'Comment';
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/create/table.md:364]:
```suggestion
COMMENT 'Comment'
```

File: docs/ru/sql-reference/statements/create/table.md [hunk 6]
```diff
@@ -346,4 +346,39 @@ SELECT * FROM base.t1;
 └───┘
 ```
 
+## COMMENT TABLE {#comment-table}
```
Comment (olgarev) [inline docs/ru/sql-reference/statements/create/table.md:349]:
```suggestion
## Секция COMMENT {#comment-table}
```

**Review by kssenii — CHANGES_REQUESTED** [review]

File: docs/en/sql-reference/statements/create/table.md [hunk 7]
```diff
@@ -354,3 +354,39 @@ SELECT * FROM base.t1;
 │ 3 │
 └───┘
 ```
+
+## COMMENT Clause {#comment-table}
+
+You can add a comment to the table when you creating it.
+
+!!!note "Note"
+    The comment is supported for all table engines except [Kafka](../../../engines/table-engines/integrations/kafka.md) and [RabbitMQ](../../../engines/table-engines/integrations/rabbitmq.md).
```
[the remark this answers is not in the document]
Comment (kirillikoff) [reply docs/en/sql-reference/statements/create/table.md:363]: I suggest adding information about the embeddedRocksDB engine. And delete the note as soon as the code is finalized.

**Review by kssenii — APPROVED** [review]

**Pull request merged**